.github/workflows/main.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        verify_flag: ["", "--vac", "--vac --horn-bmc-solver=smt-y2", "--cex", "--cex --horn-bmc-solver=smt-y2"]
+        verify_flag: ["", "--vac", "--vac --horn-bmc-solver=smt-y2", "--cex", "--cex --horn-bmc-solver=smt-y2", "klee"]
```

Comment:
split into a separate klee action. There is no need to put everything we want into a single action. This will be much easier to manage

Reply:
Do you mean to create a separate GitHub action for Klee CI? And this action will not integrate into PR? Just everyday schedule?

Reply:
1. definitely separate action for klee
2. I would start with a separate klee action that runs nighly
3. once 2. works, I would create a third action that builds the klee docker nightly, and switch 2 to use that action. Then 2 can be a regular, not cron, action